client/galaxy/scripts/mvc/tool/tool-form-base.js

```diff
@@ -46,15 +46,27 @@ define(['utils/utils', 'utils/deferred', 'mvc/ui/ui-misc', 'mvc/form/form-view',
         /** Build form */
         _buildForm: function(options) {
             var self = this;
+            var req_message = self._templateRequirements(options)[0].textContent;
             this.options = Utils.merge(options, this.options);
-            this.options = Utils.merge({
-                icon            : options.icon,
-                title           : '<b>' + options.name + '</b> ' + options.description + ' (Galaxy Version ' + options.version + ')',
-                operations      : !this.options.hide_operations && this._operations(),
-                onchange        : function() {
-                    self.refresh();
-                }
-            }, this.options);
+            if (options.requirements.length > 0) {
```

Comment:
This conditional should be used to build the title string only since the remaining parts do not depend on the conditional.